server/etcdserver/apply.go

```diff
@@ -407,6 +409,8 @@ func (a *applierV3backend) Range(ctx context.Context, txn mvcc.TxnRead, r *pb.Ra
 			sorter = &kvSortByMod{&kvSort{rr.KVs}}
 		case r.SortTarget == pb.RangeRequest_VALUE:
 			sorter = &kvSortByValue{&kvSort{rr.KVs}}
+		default:
+			lg.Panic("unexpected sort target", zap.String("SortTarget", fmt.Sprintf("%s", r.SortTarget)))
```

Comment:
```suggestion
			lg.Panic("unexpected sort target", zap.Object("sort-target", r.SortTarget))
```
Two things:
* Key names should be lowercase joined by dash "-"/
* Can we pass r.SortTarget without fmt.Sprintf?

Reply:
@serathius  Thanks for the comment. Just updated the key to "sort-target" per your comment. 

Regarding to the fmt.Sprintf, I actually thought about it previously. I can use zap.Int32 because the r.SortTarget is actually int32 ( `type RangeRequest_SortTarget int32` ). But there are two minor comments/concerns:
1. zap.Int32 is NOT used anywhere in etcd's source code; 
2. The type of r.SortTarget is RangeRequest_SortTarget, so I still need to convert the type to int32 although its real type is int32, which reduces the readability of the source code a little. 

What do you think?

Reply:
Changed to `zap.Int32("sort-target", int32(r.SortTarget))`.  Please let me know if you have any other comments or concerns.